etc/helm/pachyderm/values.yaml

```diff
@@ -210,6 +210,9 @@ pachd:
     #  expose: true
     #  port: 30650
   enterpriseLicenseKey: ""
+  enterpriseRootToken: ""
```

Comment:
Need to run `make schema` to regen the the json schema

Reply:
right! thanks